src/services/compare-appian/serverless.yml

```diff
@@ -157,14 +157,6 @@ functions:
       statusTableName: ${param:statusTableName}
       errorsTopicArn: ${param:errorsTopicArn}
       namespace: ${self:service}-${sls:stage}
-  sendDoesNotExistAlert:
-    handler: handlers/sendDoesNotExistAlert.handler
-    environment:
-      region: ${self:provider.region}
-      project: ${self:custom.project}
-      stage: ${sls:stage}
-      sesLogGroupName: ${param:sesLogGroupName}
-      errorsTopicArn: ${param:errorsTopicArn}
   sendNoMatchAlert:
```

Comment:
You will also need to change line 262 from

`            Resource: !Sub "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${self:service}-${sls:stage}-sendDoesNotExistAlert"
`

To

`            Resource: !Sub "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${self:service}-${sls:stage}-sendNoMatchAlert"
`

Reply:
Ah, great catch!